### PR TITLE
Media upload to Azure Storage

### PR DESCRIPTION
**Pull request by koxanybak**
Make the uploaded media to Strapi go to Azure Storage in production.

### CODE REVIEW

**Comment by PurkkaKoodari** [comment]
LGTM. Let's wait for #26 merge and update this before merging.

**Comment by mrnym** [comment]
Updated to v4 according to: [docs](https://github.com/jakeFeldman/strapi-provider-upload-azure-storage#usage)

File: config/plugins.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+module.exports = ({ env }) => {
+  // If 'upload' is undefined strapi will use local file system for
+  // file uploads. This is the preferred option in development mode
+  // since the database in docker containing upload URIs doesn't sync with Azure,
+  // and so if the database is reset, it will not show images that are already
+  // previously uploaded to Azure.
+  if (env('NODE_ENV') === 'development') return { upload: undefined }
```
Comment (PurkkaKoodari) [inline config/plugins.js:7]: We can move this file to `config/env/production` to avoid a manual check

**Review by PurkkaKoodari — CHANGES_REQUESTED** [review]

**Review by PurkkaKoodari — APPROVED** [review]
LGTM

**Pull request merged**